functions/index.js

```diff
@@ -1,9 +1,26 @@
 const admin = require('firebase-admin');
 const functions = require('firebase-functions');
 const express = require('express');
-
+const bodyParser = require('body-parser');
 const app = express();
 
+app.use(bodyParser.urlencoded({extended:false}));
+
+var serviceAccount = require("../ecommerce-eed77-firebase-adminsdk-dikkn-01149939c5.json");
+
+admin.initializeApp({
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: "https://ecommerce-eed77.firebaseio.com"
```

Comment:
is it possible to store database url in some config file and same for serviceAccount?

Not Important Right Now